.redhat-ci.yml

```diff
@@ -7,7 +7,7 @@ container:
     image: projectatomic/ostree-tester
 
 tests:
-    - yum -y install clang libubsan python-sphinx createrepo_c gperf python-devel 'check-devel' 'pkgconfig(ostree-1)' 'pkgconfig(libgsystem)' 'pkgconfig(librepo)' 'pkgconfig(libsolv)' 'pkgconfig(rpm)'
+    - yum -y install clang libubsan python-sphinx createrepo_c gperf python-devel 'check-devel' 'pkgconfig(ostree-1)' 'pkgconfig(libgsystem)' 'pkgconfig(librepo)' 'pkgconfig(libsolv)' 'pkgconfig(rpm)' 'pkgconfig(json-glib-1.0)'
```

Comment:
You should be able to go back to using the `packages` list now if you'd like (I see the automated build with the fix just finished up).